catalog/app/components/Preview/loaders/Html.js

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react'
+import * as AWS from 'utils/AWS'
+import AsyncResult from 'utils/AsyncResult'
+import * as Config from 'utils/Config'
+
+import { PreviewData } from '../types'
+
+import * as Text from './Text'
+import * as utils from './utils'
+
+export const detect = utils.extIn(['.htm', '.html'])
+
+const IFrameLoader = ({ handle, children }) => {
+  const sign = AWS.Signer.useS3Signer()
```

Comment:
we should note, if it's true, that unlike markdown and vega, we don't sign any URLs internal to the HTML. is that right?

Reply:
that's right